webapp/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "matterpoll",
-  "version": "1.1.0",
+  "version": "0.0.1",
```

Comment:
What do you think about just removing the version number and rely on the version in `plugin.json`?

Reply:
https://docs.npmjs.com/files/package.json#version
> If you don’t plan to publish your package, the name and version fields are optional.

Since `version` is optional, removing it is no problem.
Additionally, `0.0.1` has no means, so it should be removed. Thanks.